ctru-rs/src/services/apt.rs

```diff
@@ -90,3 +132,46 @@ impl Drop for Apt {
         unsafe { ctru_sys::aptExit() };
     }
 }
+
+/// Can launch other applications when the current one exits.
+pub struct Chainloader<'a> {
+    _apt: &'a Apt,
+}
+
+impl<'a> Chainloader<'a> {
+    /// Gets a handle to the chainloader
+    pub fn new(apt: &'a Apt) -> Self {
+        Self { _apt: apt }
+    }
+
+    /// Checks if the chainloader is set
+    #[doc(alias = "aptIsChainload")]
+    pub fn is_set(&self) -> bool {
+        // static funtion not exported
+        unsafe { (ctru_sys::envGetSystemRunFlags() & ctru_sys::RUNFLAG_APTCHAINLOAD) != 0 }
+    }
+
+    /// Clears the chainloader state.
+    #[doc(alias = "aptClearChainloader")]
+    pub fn clear(&mut self) {
+        unsafe { ctru_sys::aptClearChainloader() }
+    }
+
+    /// Configures the chainloader to launch a specific application.
+    #[doc(alias = "aptSetChainloader")]
+    pub fn set_chainloader(&mut self, title: &super::am::Title<'_>) {
```

Comment:
nit: Since these are methods on `Chainloader`, I'd probably drop `chainloader` from the method names, maybe something like:
- `Chainloader::set(title)`
- `Chainloader::set_caller()`
- `Chainloader::set_self()`